packages/jsii-kernel/lib/serialization.ts

```diff
@@ -520,6 +520,15 @@ export const SERIALIZERS: {[k: string]: Serializer} = {
         host.debug('ANY is a Ref');
         return host.objects.findObject(value).instance;
       }
+    
+      // if the value has a struct token, it was serialized by a typed jsii
+      // struct, but since the deserialization target is ANY, all we can do is
+      // strip the data from $jsii.struct and continue to deserialize as ANY.
+      if (isWireStruct(value)) {
+        const { fqn, data } = value[TOKEN_STRUCT];
+        host.debug(`ANY is a struct of type ${fqn}`);
+        return SERIALIZERS[SerializationClass.Any].deserialize(data, _type, host);
```

Comment:
Could have passed to the `SerializationClass.Struct` deserialize instead... That'd kinda feel like a better way?

Reply:
The struct deserializer assumes `type` is an FQN and crashes when called with `any`. Let me see what it entails to update it to special case `any`.

Reply:
This is getting too hairy to fix in the struct deserializer.

Reply:
Care to add a note (TODO or whatever?)

Reply:
Your advise to pass in the FQN worked, so I'll push another commit to fix.

Reply:
Oh sweet!

Reply:
Done